src/pages/experimental/order-summary-template/style.mdx

```diff
@@ -1,10 +1,54 @@
 ---
-title: Order summary template
-tabs: ['Usage', 'Style']
+title: Order summary
+tabs: ['Usage', 'Style', 'Code']
```

Comment:
```suggestion
tabs: ['Usage', 'Style', 'Code']
```

This is the only page with the Code page linked as a tab. Is that intentional?

Reply:
That was accidental. I copied the markdown from the cloud pal page and forgot to remove the code tab.

Reply:
Ah, ok. I just proposed the change to remove the code tab from this page